jsx/Form.js

```diff
@@ -1356,20 +1424,17 @@ FileElement.propTypes = {
   id: PropTypes.string,
   disabled: PropTypes.bool,
   required: PropTypes.bool,
-  hasError: PropTypes.bool,
   errorMessage: PropTypes.string,
   onUserInput: PropTypes.func,
 };
 
 FileElement.defaultProps = {
   name: '',
   label: 'File to Upload',
-  value: '',
+  value: null,
```

Comment:
not in **24.1**

_this appears to be a change that was made frequently, and should likely be adopted in **main_**

to include as **CBIG override** with plans to push to **main** in some capacity